src/fetching/getCampaignEvents.ts

```diff
@@ -1,5 +1,15 @@
+interface ZetkinEvent {
+    activity: { title: string },
+    campaign: { title: string },
+    end_time: string,
+    id: number,
+    location: { title: string },
+    start_time: string,
+    title: string
+}
```

Comment:
Wasn't this already implemented somewhere else? Instead of duplicating, I think we should split this out into a separate file and reuse the same interface in both places. Does that make sense?

Reply:
Yes, it is also used in getEvents. Sounds good!